api/v1beta1/hostedcluster_types.go

```diff
@@ -676,6 +675,18 @@ type KubevirtPlatformSpec struct {
 	// +optional
 	// +immutable
 	BaseDomainPassthrough *bool `json:"baseDomainPassthrough,omitempty"`
+	// ExtClusterKubeConfigSecret is the secret name that holds the kubeconfig for the external infra cluster
+	//
+	// +optional
+	// +immutable
+	ExtClusterKubeConfigSecret *KubeconfigSecretRef `json:"kubeConfigSecret,omitempty"`
```

Comment:
the json portion of this probably needs a more descriptive name than `kubeConfigSecret`.

Reply:
changed to `extClusterKubeConfigSecret`